mcs/class/referencesource/System.Data.Entity/System/Data/Query/PlanCompiler/TransformationRules.cs

```diff
@@ -1017,7 +1017,7 @@ private static bool ProcessSimplifyCase_EliminateWhenClauses(RuleProcessingConte
         ///     WHEN W1 THEN T1 
         ///     WHEN W2 THEN T2 ... 
         ///     ELSE (CASE 
-        ///             WHEN WN1 THEN TN1, � 
+        ///             WHEN WN1 THEN TN1, � 
```

Comment:
Looks like the file was in Windows-1252 originally (that should be a '…' there). Please convert it to utf-8.

Reply:
![image](https://user-images.githubusercontent.com/1798510/73120372-72e7cb00-3f6d-11ea-8bd8-4fac4f0ca99a.png)

Reply:
@lambdageek I made sure it is UTF-8. I delete that weird character in order to fix it.